contrib/cli/great_expectations_contrib/commands.py

```diff
@@ -177,7 +178,8 @@ def write_package_to_disk(
         path: The relative path to the target package JSON file.
     """
     json_dict = package.to_json_dict()
-    data = json.dumps(json_dict, indent=4)
+    json_dict_serialized = convert_to_json_serializable(json_dict)
```

Comment:
We need to make sure this is json serializable using our custom method that handles complex types before serializing using the built in json.dumps.